src/views/Balance.vue

```diff
@@ -1,15 +1,37 @@
 <template>
     <h2 class="text-gray-400 text-3xl font-medium">Balance</h2>
+    <span class="text-gray-400">{{ balance ? balance.toString() : '0' }}</span>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
-// import { store } from '../store';
+import BN from 'bn.js';
+import { defineComponent, ref, watch } from 'vue';
+import keyring from '@polkadot/ui-keyring';
+import { subscribeBalance } from '@/api/query/accountInfo';
+import { Balance } from '@/api/models';
+import { store } from '@/store';
 
 export default defineComponent({
-    async setup() {
-        // const api = await store.getters.api;
-        return {};
+    setup() {
+        const balance = ref<Balance>(new BN(0));
+        // TODO: move keyring to store;
+        let accounts;
+        // TODO: map multiple unsubs;
+        watch(
+            () => store.state.initialized,
+            () => {
+                if (store.getters.initialized) {
+                    accounts = keyring.getPairs();
+                    // TODO: switch accounts
+                    const account = accounts[0];
+                    subscribeBalance(account.address, balance);
```

Comment:
I'm getting an `Unhandled error during execution of watcher callback` error alongside `TypeError: Cannot read property 'address' of undefined`. This is resulting in the page not being changed when clicking on the menu like before (i.e., cannot mount a different component).
Full error log:
```
[Vue warn]: Unhandled error during execution of watcher callback 
  at <Balance onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy {…}[[Handler]]: Objectget: ƒ get({ _: instance }, key)has: ƒ has({ _: { data, setupState, accessCache, ctx, appContext, propsOptions } }, key)ownKeys: (target) => {…}set: ƒ set({ _: instance }, key, value)__proto__: Object[[Target]]: Objectbalance: (...)$: (...)$attrs: (...)$data: (...)$el: (...)$emit: (...)$forceUpdate: (...)$nextTick: (...)$options: (...)$parent: (...)$props: (...)$refs: (...)$root: (...)$router: (...)$slots: (...)$store: (...)$watch: (...)_: (...)get balance: () => setupState[key]set balance: () => { }get $: () => publicPropertiesMap[key](instance)set $: () => { }get $attrs: () => publicPropertiesMap[key](instance)set $attrs: () => { }get $data: () => publicPropertiesMap[key](instance)set $data: () => { }get $el: () => publicPropertiesMap[key](instance)set $el: () => { }get $emit: () => publicPropertiesMap[key](instance)set $emit: () => { }get $forceUpdate: () => publicPropertiesMap[key](instance)set $forceUpdate: () => { }get $nextTick: () => publicPropertiesMap[key](instance)set $nextTick: () => { }get $options: () => publicPropertiesMap[key](instance)set $options: () => { }get $parent: () => publicPropertiesMap[key](instance)set $parent: () => { }get $props: () => publicPropertiesMap[key](instance)set $props: () => { }get $refs: () => publicPropertiesMap[key](instance)set $refs: () => { }get $root: () => publicPropertiesMap[key](instance)set $root: () => { }get $router: () => globalProperties[key]set $router: () => { }get $slots: () => publicPropertiesMap[key](instance)set $slots: () => { }get $store: () => globalProperties[key]set $store: () => { }get $watch: () => publicPropertiesMap[key](instance)set $watch: () => { }get _: () => instance__proto__: Object[[IsRevoked]]: false > > 
  at <RouterView> 
  at <DashboardLayout> 
  at <KeepAlive> 
  at <App>
```

Reply:
This error occurs when there are no accounts loaded in the extension